crates/core/component/stake/src/validator/bonding.rs

```diff
@@ -44,12 +46,14 @@ impl From<State> for pb::BondingState {
             state: match v {
                 State::Bonded => pb::bonding_state::BondingStateEnum::Bonded as i32,
                 State::Unbonded => pb::bonding_state::BondingStateEnum::Unbonded as i32,
-                State::Unbonding { unbonding_epoch: _ } => {
-                    pb::bonding_state::BondingStateEnum::Unbonding as i32
-                }
+                State::Unbonding {
+                    unbonds_at_epoch: _,
+                } => pb::bonding_state::BondingStateEnum::Unbonding as i32,
```

Comment:
```suggestion
                State::Unbonding {
                    ..
                } => pb::bonding_state::BondingStateEnum::Unbonding as i32,
```

cosmetic. not a blocker.